apps/docs/src/common/pages/get-started/prototype.tsx

```diff
@@ -0,0 +1,66 @@
+import { FC, Fragment, createElement as h } from 'react';
+import { PageProps } from '@not-govuk/app-composer';
+import { A } from '@not-govuk/components';
+
+
+
+const Page: FC<PageProps> = ({ location }) => (
+  <div className="govuk-grid-row">
+    <div className="govuk-grid-column-one-third">
+      <aside>
+        <h2>Get started</h2>
+          <ul className="plain">
+            <li><A href="/get-started/prototype">Prototyping for designers</A></li>
+            <li><A href="/get-started/project">Starting a project for developers</A></li>
+          </ul>
+      </aside>
+    </div>
+
+    <div className="govuk-grid-column-two-thirds">
+      <h1>
+        <span className="caption">Get started</span>
+        Prototyping for designers
+      </h1>
+        <p>This guide explains how to quickly build prototypes to test ideas with users.</p>
```

Comment:
How about?:
> This guide will explain how to quickly build a standalone prototype, outside of an established project. Prototypes are typically used by UCD specialists to test ideas with users.
> 
> A standalone prototype is useful in the following scenarios:
> 1. Designers working on projects that are not currently using the HODS production framework
> 2. Designers who wish to work independently on ideas
> 3. Developers who wish to try quickly out the system
>
> Note: If you are a designer working on a project which _is_ using the HODS production framework you might want to consider having the developers create you a blank 'app' within the project as this will make it easier to eventually transform your prototype into a production application. Although this down depend on your teams ways of working.

Reply:
I don't know, maybe that's too long winded.

See what you think and just use the bits (if any) you think are useful.

Reply:
This is quite important to know. What about:

> There are two different ways to set up the Home Office prototype, depending on whether your team uses the HODS production framework for its build:
> 
> - blank ‘app’ within the development project - if your project uses the HODS production framework, you might consider having the developers create a blank 'app' within the project for you to make it easier to transform your prototype into a production application

> - ‘standalone’ prototype - if your project is not using the HODS production framework but you want to quickly try out the system to test ideas with users, you can set this up yourself by following the steps below